Userland/Libraries/LibGL/GLContext.cpp

```diff
@@ -736,6 +749,17 @@ void GLContext::gl_disable(GLenum capability)
     bool update_rasterizer_options = false;
 
     switch (capability) {
+    case GL_CLIP_PLANE0:
+    case GL_CLIP_PLANE1:
+    case GL_CLIP_PLANE2:
+    case GL_CLIP_PLANE3:
+    case GL_CLIP_PLANE4:
+    case GL_CLIP_PLANE5: {
+        auto pl_idx = static_cast<size_t>(capability) - GL_CLIP_PLANE0;
```

Comment:
```suggestion
        auto plane_idx = static_cast<size_t>(capability) - GL_CLIP_PLANE0;
```

Reply:
Thanks for the warm welcome! These items have been changed in the latest commit.